manifests/config.pp

```diff
@@ -105,11 +114,163 @@
   } else {
     $_java_opts = $java_opts
   }
-  file_line { 'standalone.conf-JAVA_OPTS':
+  file_line { 'JAVA_OPTS':
```

Comment:
This filename needs to somehow be specific to Keycloak to avoid potential conflicts with other modules or other Puppet code.  So maybe change to `keycloak-JAVA_OPTS`